src/pages/transactions/index.tsx

```diff
@@ -119,6 +126,15 @@ export const requestTransactionsDefault = async (
             transaction.precision =
               assetPrecisions[contract.parameter.currencyID];
           }
+          if (contract?.type === TransactionType.Claim) {
```

Comment:
same here, also, is it necessary to call findReceipt with same args 2 times in the same page?